ipa-core/src/protocol/ipa_prf/aggregation/step.rs

```diff
@@ -0,0 +1,36 @@
+use ipa_step_derive::CompactStep;
+
+#[derive(CompactStep)]
+pub(crate) enum AggregationStep {
+    #[step(child = BucketStep)]
+    MoveToBucket,
+    #[step(count = 32, child = AggregateValuesStep)]
+    Aggregate(usize),
+}
+
+#[derive(CompactStep)]
+pub enum BucketStep {
+    /// should be equal to `MAX_BREAKDOWNS`
+    #[step(count = 512, child = crate::protocol::boolean::step::BoolAndStep)]
```

Comment:
```suggestion
    #[step(count = 512, child = crate::protocol::boolean::step::BoolAndStep), name = "b"]
```

@benjaminsavage was sad that this said "bit" rather than "bucket", so "bucket" is also fine.  I just prefer to use shorter names when possible.

We could also make this `struct BucketStep(usize)`, which could be even shorter.